examples/vnet/doca_flow.c

```diff
@@ -64,10 +63,26 @@ int
 doca_flow_pipe_rm_entry(uint16_t pipe_queue,
 			struct doca_flow_pipe_entry *entry)
 {
-	DOCA_LOG_INFO("(pipe %d) HW release id%d", pipe_queue, entry->id);
-	/* doca_dpdk_free_flow(0, entry->pipe_entry); */
-	free(entry);
-	return 0;
+	struct doca_flow_pipe *pipe;
+	int ret = 0;
+
+	if (entry == NULL || entry->flow_pipe == NULL)
+		return -EINVAL;
+	pipe = (struct doca_flow_pipe *)entry->flow_pipe;
+	if (doca_dpdk_find_pipe_entry(pipe_queue, pipe, entry)) {
```

Comment:
if entry list pointers are not NULL or trahsed.

Reply:
Done.